packages/fx-core/src/plugins/resource/bot/configs/scaffoldConfig.ts

```diff
@@ -72,22 +77,27 @@ export class ScaffoldConfig {
     // TODO: support other stages (maybe addCapability)
     const fromInputs = context.answers?.stage === Stage.create;
     if (fromInputs) {
-      // TODO: retrieve host type from context.answers
-      // Since the UI design is not finalized yet,
-      // for testing purpose we currently use an environment variable to select hostType.
-      // Change the logic after question model is implemented.
-      if (process.env.TEAMSFX_BOT_HOST_TYPE) {
-        return process.env.TEAMSFX_BOT_HOST_TYPE === "function"
-          ? HostTypes.AZURE_FUNCTIONS
-          : HostTypes.APP_SERVICE;
-      } else {
-        return undefined;
-      }
+      return context.answers
```

Comment:
Nit: possible to use something like `return context.answers ?? value`?

Reply:
`context.answers ?? value` is equivalent to `context.answers ? context.answers : value` which is a different logic from the current code.